packages/common/src/controllers/AccountController.ts

```diff
@@ -409,22 +386,30 @@ export default class AccountController {
     return !!responseData?.accessGranted;
   };
 
-  reloadActiveSubscription = async ({ delay }: { delay: number } = { delay: 0 }): Promise<unknown> => {
+  reloadSubscriptions = async ({ delay }: { delay: number } = { delay: 0 }): Promise<unknown> => {
     useAccountStore.setState({ loading: true });
 
     const { getAccountInfo } = useAccountStore.getState();
     const { customerId } = getAccountInfo();
+    const { accessModel } = useConfigStore.getState();
 
     // The subscription data takes a few seconds to load after it's purchased,
     // so here's a delay mechanism to give it time to process
     if (delay > 0) {
       return new Promise((resolve: (value?: unknown) => void) => {
         setTimeout(() => {
-          this.reloadActiveSubscription().finally(resolve);
+          this.reloadSubscriptions().finally(resolve);
         }, delay);
       });
     }
 
+    // For non-SVOD platforms, there could be TVOD items, so we only reload entitlements
+    if (accessModel !== 'SVOD') {
+      await this.refreshEntitlements?.();
+
+      return useAccountStore.setState({ loading: true });
```

Comment:
This also looks out of place. If it is supposed to be here, please add a comment as to why.

Reply:
👀